README.md

```diff
@@ -20,55 +29,46 @@ frameworks to mount and unmount Lustre filesystems to/from containers in their p
 
 ## Deployment
 
+This describes methods of deploying the Lustre CSI driver in various environments.
+
 ### Helm
 
 You can use Helm to manage the lustre CSI driver components:
 - To deploy: `cd charts/ && helm install lustre-csi-driver lustre-csi-driver/ --values lustre-csi-driver/values.yaml`
 - To shut down: `helm delete lustre-csi-driver`
 
-### Kind
-
-Assuming the nnf-sos kind cluster is created...
-
-Deploy the {lustre,mock} DaemonSet and CSIDriver on the NNF Nodes
-
-```bash
-./node.sh create {lustre,mock}
-```
+### Kubernetes
 
-## Examples
+Deployment uses [Kustomize](https://kustomize.io/) to configure the deployment YAMLs in the [kustomization base](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays) 
+[deploy/kubernetes/base](./deploy/kubernetes/base).
+- To deploy using the Makefile: `make deploy`
+- To undeploy using the Makefile: `make undeploy`
 
-Deploy an example mock pod with volume mount /mnt/nnf
+To deploy a specific [overlay](./deploy/kubernetes/overlays):
+- `make deploy OVERLAY=overlays/<overlay>`
 
-```bash
-kustomize build config/examples/mock | kubectl create -f -
-```
+Otherwise, you can just use the pre-built .yaml files in [deploy/kubernetes](./deploy/kubernetes):
+- `kubectl apply -f https://raw.githubusercontent.com/HewlettPackard/lustre-csi-driver/master/deploy/kubernetes/lustre-csi-driver.yaml`
 
-## Manual Testing
-
-Start NNF Driver:
-
-```bash
-CSI_ENDPOINT=tcp://127.0.0.1:10000 ./nnf-csi-driver
-```
-
-Test NNF Driver using csc: `https://github.com/rexray/gocsi/tree/master/csc`
-
-Get plugin info:
+### Kind
 
-```bash
-csc identity plugin-info --endpoint tcp://127.0.0.1:10000
-"nnf-csi-driver" "v0.0.1"
-```
+This assumes your [Kind](https://kind.sigs.k8s.io/) environment is already set up and ready for a deployment.
+A Kind [kustomization overlay](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays) is defined by the YAMLs in [deploy/kubernetes/overlays/kind](./deploy/kubernetes/overlays/kind).
+- To deploy using the Makefile: `make kind-push && make kind-deploy`
+- To undeploy using the Makefile: `make kind-undeploy`
 
-### NodePublish
+## Usage
 
-```bash
-csc node publish --cap ACCESS_MODE,ACCESS_TYPE[,FS_TYPE,MOUNT_FLAGS] --target-path TARGET_PATH VOLUME_ID [VOLUME_ID...]
-```
+This section provides examples for consuming a Lustre filesystem via a Kubernetes [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) 
+(PV) and [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#lifecycle-of-a-volume-and-claim) (PVC), 
+and finally an example of using the PVC in a simple application deployed as a Pod. 
 
-Example
+It assumed that a Lustre filesystem is already created and mounted on the underlying host OS, and that the Lustre CSI
```

Comment:
Strike "and mounted on the underlying host OS".  That's not necessary.

Reply:
Done